templates/app/views/users/edit.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= render 'shared/error_messages', target: @user %>
 
   <div class="col-span-full space-y-4">
-    <h1 class="font-serif text-h4 md:text-h3">My account</h1>
+    <h1 class="font-serif text-h4 md:text-h3"><%= I18n.t('spree.my_account') %></h1>
```

Comment:
I think there was some discussion in Slack, but let's consistently use the `t` helper in views. It is strictly more powerful than calling `I18n`.

Reply:
I honestly would be in favor of slinging them all over to that format and afterwards documenting properly how to change them.